tesseract_core/sdk/cli.py

```diff
@@ -449,8 +478,47 @@ def serve(
     else:
         ports = None
 
+    if no_compose:
```

Comment:
This branch could be moved to `engine.py`?

Reply:
no strong opinions on this from my side